app/controllers/teachers/classroom_manager_controller.rb

```diff
@@ -39,21 +31,11 @@ def generic_add_students
   end
 
   def retrieve_classrooms_for_assigning_activities # in response to ajax request
-    classrooms_and_their_students = current_user.classrooms_own.map do |classroom|
-      {  classroom: classroom, students: classroom.students.sort_by(&:sorting_name)}
-    end
-    render json: {
-      classrooms_and_their_students: classrooms_and_their_students
-    }
+    render json: classroom_with_students_json(current_user.classroom_own)
```

Comment:
Do we need to change this to classrooms_i_own? And does this mean we can un-comment the test that fails due to this error?

Reply:
Yes changing this to classrooms I own should make the test pass